### ISSUE TITLE
De novo design of small-molecule–induced conformational change

### ISSUE THREAD

**Issue by delalamo** [delalamo/SKM#636]
```bibtex
@misc{chang2026,
  title = {De novo design of small-molecule-induced conformational change},
  author = {Chang, Jeffrey and Polizzi, Nicholas F.},
  year = {2026},
  doi = {10.64898/2026.08.03.742366},
  url = {https://doi.org/10.64898/2026.08.03.742366},
  note = {bioRxiv, Version 1}
}
```

**Comment by delalamo** [comment]
#### Candidate atomic notes

- **A mobile lid can convert a static de novo small-molecule binder into a ligand-induced conformational switch** [@chang2026]. Figure 2A–B, Figure S1, and Table S4 identify ligand-dependent changes in 4 of 24 designs by fluorescence anisotropy and 9 of 24 by limited proteolysis, with two leading designs supported by both assays. The evidence is moderately strong because it combines orthogonal solution measurements, but it comes from one starting binder–ligand system; moreover, the ligand-bound crystal structures in Figure 3 do not provide direct apo/holo structural pairs for the leading designs. This supports [[Modifying conformational landscapes of proteins can give rise to new functions]], while the absence of direct structural observation of their open states limits the mechanistic claim. Related: [[Modifying conformational landscapes of proteins can give rise to new functions]]; [[Protein dynamics]]; [[Conformational entropy can decrease or increase following ligand binding]]. Placement: new atomic note, `A mobile lid can convert a static protein binder into a ligand-induced conformational switch.md`.

- **Ligand-induced closure can increase small-molecule specificity more than it increases cognate affinity** [@chang2026]. Figures 2D–E and 3 and Figure S4 show that two lidded designs bind the cognate ligand at approximately 0.2 and 0.5 nM, compared with 1.2 nM for the parent, while discrimination against a close analog improves by more than 100-fold; their ligand-bound crystal structures closely match the predictions and show complete ligand burial. This is strong structural and biophysical evidence for two designs, but specificity was tested against only one close analog. The much larger change in discrimination than in cognate affinity suggests that the lid’s principal functional contribution is negative selection against an alternative ligand, not simply stronger binding. This complicates a simple reading of [[Dynamics and specificity are inversely related]] because a dynamic apo state followed by ligand-dependent ordering produced greater specificity. Related: [[Dynamics and specificity are inversely related]]; [[Conformational entropy can decrease or increase following ligand binding]]; [[De novo structure prediction with ligands outperforms apo de novo structure prediction alone]]. Placement: new atomic note, `Ligand-induced closure can increase specificity more than cognate affinity.md`.

- **Conformational gating can tune small-molecule association kinetics without abolishing high-affinity binding** [@chang2026]. Figures 2F and 4, Figures S8–S9, and Table S7 show association rates spanning roughly two orders of magnitude; one targeted variant increased association approximately thirtyfold while retaining subnanomolar affinity. The evidence for association-rate tuning is strong across two orthogonal fluorescence measurements, but the reported multi-month residence time is inferred from measured affinity and association rate rather than observed through a direct dissociation experiment. That distinction materially weakens the strongest kinetic claim and should remain explicit. Related: [[Protein dynamics]]; [[Conformational entropy can decrease or increase following ligand binding]]; [[Dynamics and specificity are inversely related]]. Placement: new atomic note, `Conformational gating can tune small-molecule association kinetics without abolishing affinity.md`.

- **Ligand-dependent AlphaFold3 confidence can nominate conformational switches but does not validate their apo ensembles** [@chang2026]. Figure 1D, Figure S2, and Table S3 show that relative confidence between ligand-bound and ligand-free predictions was used to select candidates from a large computational set, after which several showed experimental ligand responsiveness. This is moderate evidence for candidate nomination, but no rejected or randomly selected designs were tested, so enrichment, sensitivity, and specificity cannot be estimated. Figure S6 supplies a direct caution: an apo crystal structure remained closed where AlphaFold3 predicted a register-shifted lid. This supports, rather than overturns, existing warnings that confidence metrics and single-structure predictors do not determine conformational populations or energetics. Related: [[pLDDT and PAE inversely correlated with protein dynamics in dynamic naturally occurring proteins, but not de novo proteins]]; [[Protein structure prediction methods are unable to predict the energetics of a conformational landscape unless explicitly trained for that purpose]]; [[Structure prediction methods undersample the conformational space they find to be high-confidence]]. Placement: new atomic note, `Ligand-dependent AlphaFold3 confidence can nominate but not validate conformational switches.md`.

- **Fluorescence anisotropy and limited proteolysis report different subpopulations of a designed conformational ensemble** [@chang2026]. Supplementary Text S2–S3, Figure 2A–B, Figure S1, and Table S4 show that fluorescence anisotropy identified four responsive designs whereas limited proteolysis identified nine; the supplement attributes this difference to sensitivity to local fast motions versus rare, larger opening events. The evidence is moderate because both measurements are indirect, and thermodynamic interpretation of proteolysis depends on a simplified kinetic model described in Figure S5. The differing hit sets should therefore be interpreted as complementary information about motion on different scales rather than contradictory classifications. Related: [[Protein dynamics]]; [[Conformational entropy can decrease or increase following ligand binding]]; [[Fast dynamics of proteins drive slower conformational changes]]. Placement: append under `#### Measurement` in [[Protein dynamics]].

- **A designed ligand-induced conformational change can be coupled directly to a genetically encoded FRET readout** [@chang2026]. Figure 5 and Figure S12 show a ligand-dependent FRET-ratio change, a fitted affinity of 1.5 ± 0.3 nM, and a retained response when the ligand was spiked into saliva, urine, and serum. This is moderate proof-of-concept evidence from one sensor and technical replicates; it does not include cellular measurements, patient samples, longitudinal stability, or a broad interference panel. The biological-fluid experiments demonstrate matrix compatibility, not clinical validation, despite the paper’s discussion of clinical monitoring. Related: [[Protein dynamics]]; [[Modifying conformational landscapes of proteins can give rise to new functions]]; [[Selectivity, specificity, and conformational constraints can be encoded in loss functions during protein design]]. Placement: new atomic note, `Ligand-induced conformational change can be coupled to a genetically encoded FRET readout.md`.

#### Metadata check

Full text was verified from the complete 50-page PDF, including supplementary text, Figures S1–S14, and Tables S1–S8. The issue contains a complete BibTeX entry with citekey `chang2026` and DOI `10.64898/2026.08.03.742366`. Neither the citekey nor DOI is currently present in `bibliography.bib`; the entry would need to be added before creating notes. This is bioRxiv version 1, posted August 4, 2026, licensed CC BY 4.0, and not peer reviewed. The authors disclose a provisional patent application covering the reported designs and functions. Operational design details, sequences, construct specifications, screening parameters, and experimental procedures are intentionally omitted from this summary.
